### PR TITLE
Release 1.10.38

### PR DESCRIPTION
**Pull request by IshentRas**
- Cadvisor
- Skip nodes when applying labels or schedulibilty

### CODE REVIEW

**Comment by ianmiell** [comment]
Tests pass. LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Openshift 3 Cookbook CHANGELOG
 This file is used to list changes made in each version of the Openshift 3 cookbook.
 
+## v1.10.38
+### Improvement
+- Give the possibility of adding cAdvisor port and read-only port for kubelet arguments
+- Give the possibility of skip nodes when applying schedulability and labelling
```
Comment (jperville) [inline CHANGELOG.md]: skipping nodes

File: README.md [hunk 2]
```diff
@@ -455,6 +455,37 @@ In general, override attributes in the environment should be used when changing
       ]
 ...
 ```
+* EXCLUDE NODES FROM SCHEDULING AND LABELLING("skip_run" must be define and the node will be excluded when enforcing labels and schedulability")
```
Comment (jperville) [inline README.md]: must be defined

File: CHANGELOG.md (hunk 1 above)
Comment (IshentRas) [inline CHANGELOG.md]: thanks :-) Shakespeare :+1:  lol

File: README.md (hunk 2 above)
Comment (IshentRas) [inline README.md]: :+1:

**Pull request merged**